financial-policy.md

```diff
@@ -0,0 +1,69 @@
+# denhac Financial Policy
+>[!NOTE]
+>Current Version Approved by The Board on December 6, 2022
+
+## How We Collect Money
+
+### Membership Dues:
+When a member registers for a denhac account, they will save a card on file with our card processor, Stripe. Under our Membership Agreement, a Membership is not considered active until an ID check has been performed. The member will be charged using the card on file with Stripe at the time of Access Card Activation. Membership dues are collected using Stripe on a monthly recurring basis. If we cannot charge the card on file for whatever reason, the member will be notified by Stripe and given four days to update their information. During this time, the member will still have access to the space, but after the failed attempt to charge their card, their membership will be terminated. The member can immediately update their form of payment after their membership termination and purchase a new membership.
```

Comment:
> be notified by Stripe

I'd just say notified or notified via email if you have to specify. Technically WooCommerce does the notifying, Stripe doesn't.

> given four days

I can change this to be whatever retry policy and frequency you want, but it's currently set to 4 days. We weren't sure if each failed charge might cause an overdraft or not. It tries after a half day, then a day, then another day, then fails.

> after the failed attempt

I'd say after the final attempt? It fails each time.